README.org

```diff
@@ -116,12 +116,18 @@ When a PML file is successfully analysed, any drugs found in ~requires~ blocks
 are reported back to the user as 'identified drugs'.
 
 Again, this feature can be manually tested by uploading a file.
-~panacea/test/fixtures/drugs.pml~ is a well-structured PML document containing
+~panacea/test/fixtures/no_ddis.pml~ is a well-structured PML document containing
 some drug identifiers.
 
 Automated tests for the parser and web interface also ensure that this feature
 works as intended.
 
+**** Identifying DDIs
+The drugs identified in the PML file are sent to Asclepius to check for DDIs.
+Therefore, to test this feature you must be running Asclepius: ~docker run -p "5000:5000" tomtoothfairies/asclepius~
+Automated tests confirm that this feature works as intended. You can run the tests
+specific to this feature by running ~mix test --only identify_ddis~ within the ~panacea~ directory.
```

Comment:
the `mix test` command won't work locally on his machine because everything is installed inside docker.
something like:
`docker run -t -e "MIX_ENV=test" panacea mix test --only identify_ddis`
should do the trick